js/ui/drop_down_button.js

```diff
@@ -424,6 +424,10 @@ const DropDownButton = Widget.inherit({
     },
 
     toggle(visible) {
+        if(!windowUtils.hasWindow()) {
```

Comment:
Let's get rid of this and use hasWidow() check here:
https://github.com/DevExpress/DevExtreme/pull/11847/files#diff-cc46f04faa146e60076fc47eff5a47c1R309

Reply:
fixed